src/wp-includes/html-api/class-wp-html-processor.php

```diff
@@ -1141,21 +1141,17 @@ public function step( $node_to_process = self::PROCESS_NEXT_NODE ): bool {
 	 * Breadcrumbs start at the outermost parent and descend toward the matched element.
 	 * They always include the entire path from the root HTML node to the matched element.
 	 *
-	 * @todo It could be more efficient to expose a generator-based version of this function
```

Comment:
Is this `@todo` no longer applicable?

Reply:
Correct, the comment is about the previous implementation where the breadcrumbs were calculated on the fly. That's no longer the case and the comment is stale.